quill/templates/quill/widget.html

```diff
@@ -21,8 +21,8 @@
 	var toolbarSelector = '[data-quill="{{ "toolbar_selector"|quill_conf }}"]';
 	var id = '{{ id }}';
     var imageModuleOpts = {
-        handler: '{% url "admin:quill-file-upload" %}',
-        allowedExtensions: JSON.parse('{{ "allowed_image_extensions"|quill_conf_json }}')
+        allowedExtensions: JSON.parse('{{ "allowed_image_extensions"|quill_conf_json }}'),
+        // handler: '{% url "admin:quill-file-upload" %}', #Commented this our because trown an error.
```

Comment:
This isn't a viable solution, without an explanation.  Commenting out code isn't a preferred practice either.